### PR TITLE
Feature logan signup 26

### PR DESCRIPTION
**Pull request by logan858**
### What this PR does (required):
- Updated our login & signup pages to reflect the mockup UI.

### Screenshots / Videos (required):
image
image

### Any information needed to test this feature (required):
- Feature should work as it has previously for testing, no additional requirements needed.

### Any issues with the current functionality (optional):
- No other issues, there is a space under the login button that will contain the demo user login button from ticket #18

### CODE REVIEW

File: client/.eslintrc [hunk 1]
```diff
@@ -18,6 +18,8 @@
    "react/react-in-jsx-scope": "off",
    "react/prop-types": "off",
    "react-hooks/rules-of-hooks": "error",
-   "react-hooks/exhaustive-deps": "error"
+   "react-hooks/exhaustive-deps": "error",
+   // line formatting => comment out if causing problems
```
Comment (ezeilo-su) [inline client/.eslintrc:22]: This comment is not requred

**Review by ezeilo-su — COMMENTED** [review]
💪🏻

File: client/.eslintrc (hunk 1 above)
Comment (logan858) [reply client/.eslintrc]: Removed comment 👍

File: client/src/components/AuthHeader/useStyles.ts [hunk 2]
```diff
@@ -1,34 +1,27 @@
...
     display: 'flex',
     flexWrap: 'wrap',
+    fontFamily: 'Poppins-Bold',
   },
   accAside: {
-    fontSize: 14,
-    color: '#b0b0b0',
-    fontWeight: 400,
+    fontSize: 13,
+    fontFamily: 'Poppins-Bold',
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/useStyles.ts:14]: Please set the font family within MUI's theme and use the theme to apply the font family here (avoid defining the font family directly to ensure consistent design)

File: client/src/pages/Login/Login.tsx [hunk 3]
```diff
@@ -51,6 +50,7 @@ export default function Login(): JSX.Element {
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
```
Comment (rajivtitus) [inline client/src/pages/Login/Login.tsx:53]: Can we please rename this `AuthHeader` component here (it maybe misleading sometimes as it is not a header) 😄

File: client/src/pages/Login/LoginForm/useStyles.ts [hunk 4]
```diff
@@ -6,18 +6,24 @@ const useStyles = makeStyles((theme) => ({
...
     paddingLeft: '5px',
+    textTransform: 'uppercase',
+    color: 'black',
+    fontFamily: 'Poppins-Bold',
   },
   inputs: {
     marginTop: '.8rem',
     height: '2rem',
     padding: '5px',
+    fontFamily: 'Poppins-Regular',
```
Comment (rajivtitus) [inline client/src/pages/Login/LoginForm/useStyles.ts:19]: Same as comment above for font family

**Review by rajivtitus — APPROVED** [review]
Good work - the UI looks good 👍🏽 

Just some minor comments to be addressed before merging. Thanks!

File: client/src/components/AuthHeader/useStyles.ts (hunk 2 above)
Comment (logan858) [reply client/src/components/AuthHeader/useStyles.ts]: updated themes with custom typography choices

File: client/src/pages/Login/Login.tsx (hunk 3 above)
Comment (logan858) [reply client/src/pages/Login/Login.tsx]: renamed to AuthLinks

**Pull request merged**